Gruntfile.js

```diff
@@ -1,23 +1,32 @@
 module.exports = function (grunt) {
+
+    var radarName = grunt.option('radar') || 'demo';
+    var distDir = 'radars/'+radarName+'/dist';
+
+    console.log("\nBuilding site for radar: "+radarName+"\n");
+
     grunt.initConfig({
         pkg: grunt.file.readJSON('package.json'),
-        // order matters (concatenation respects order and that order is relevant during parsing)
-        scss_sources: ['app/sass/techradar.scss'],
-        js_view_sources: ['app/javascript/init.js', 'app/javascript/techradar.js'],
+        scss_sources: ['app/sass/techradar.scss', 'radars/'+radarName+'/sass/radar.scss' ],
+        js_view_sources: ['app/javascript/init.js', 'app/javascript/techradar.js' , 'radars/'+radarName+'/javascript/*.js'],
+        image_sources: ['app/images/*', 'radars/'+radarName+'/images/*'],
```

Comment:
Where can I see/find a usage of images along with the radar - the referenced subfolder is empty?